### ISSUE TITLE
Wrong indent for nested table literals in a RHS function call

### ISSUE THREAD

**Issue by golopot** [Koihik/LuaFormatter#120]
Expected:
```lua
local a = cjson.encode(
    {
        error = {
            code = "foooooooooooooooooo",
            message = "barrrrrrrrrrrrrrrrrrrrrrr"
        }
    }
)

```

Actual:
```lua
local a = cjson.encode(
              {
        error = {
            code = "foooooooooooooooooo",
            message = "barrrrrrrrrrrrrrrrrrrrrrr"
        }
    }
          )
```

Config:
```
keep_simple_control_block_one_line: false
keep_simple_function_one_line: false
break_after_functioncall_lp : true
break_before_functioncall_rp : true
align_args: false
align_parameter: false
chop_down_parameter: true
align_table_field: false
```

**Comment by Koihik** [comment]
I can't reproduce this issue with latest code.
@golopot Are you using the extension of vscode?

**Comment by golopot** [comment]
Yes I am using the vscode extension of version 1.3.3 .

**Comment by Koihik** [comment]
Released vscode-lua-format 1.3.4 that fixed this issue.

**Comment by golopot** [comment]
Fixed after upgrading. Thanks!
